jbx-protocol/juice-interface
# Nft redemptions modal

## What does this PR do and why?

### to test

1. enable feature flag `jb.enableFeatureFlag('nftRedeem')`
2. buy an nft here http://localhost:3000/v2/p/146
3. Go to new NFT section and click "redeem for ETH"
4. Try to redeem
5. Check if you get your money back
